modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -91,15 +91,29 @@ pullSecret: '<pull_secret>'
 sshKey: '<ssh_pub_key>'
 ----
 +
+--
 <1> Scale the worker machines based on the number of worker nodes that are part of the {product-title} cluster. Valid options for the `replicas` value are `0` and integers greater than or equal to `2`. Set the number of replicas to `0` to deploy a three-node cluster, which contains only three control plane machines. A three-node cluster is a smaller, more resource-efficient cluster that can be used for testing, development, and production. You cannot install the cluster with only one worker.
 <2> When deploying a cluster with static IP addresses, you must set the `bootstrapExternalStaticIP` configuration setting to specify the static IP address of the bootstrap VM when there is no DHCP server on the `baremetal` network.
 <3> When deploying a cluster with static IP addresses, you must set the `bootstrapExternalStaticGateway` configuration setting to specify the gateway IP address for the bootstrap VM when there is no DHCP server on the `baremetal` network.
 <4> See the BMC addressing sections for more options.
 <5> Set the path to the installation disk drive, for example, `/dev/disk/by-id/wwn-0x64cd98f04fde100024684cf3034da5c2`.
++
+[IMPORTANT]
+====
+For Ironic, to set the path to the installation disk drive, enter the kernel name of the disk. For example `/dev/sda`. Alternatively, to set the disk WWN, replace the `deviceName` parameter with the `wwnWithExtension` parameter.
```

Comment:
Ironic is the underlying technology of the OpenShift IPI installer for bare-metal, it is not a special case.

```suggestion
Because the disk discovery order is not guaranteed, the kernel name of the disk can change across boots for machines with multiple disks (e.g. `/dev/sda` becomes `/dev/sdb` and vice-versa).
To avoid this issue, it is recommended to use persistent disk attributes like its WWN. To use the disk WWN, replace the `deviceName` parameter with the `wwnWithExtension` parameter and the disk name with the disk WWN, e.g. `"0x64cd98f04fde100024684cf3034da5c2"` (quoting is important because it must be used as a string value and not an hexadecimal value).
```